test/internal/k8sutils/utils_get.go

```diff
@@ -43,9 +43,11 @@ func GetPodsIpsByNode(ctx context.Context, clientset *kubernetes.Clientset, name
 	if err != nil {
 		return nil, err
 	}
-	ips := make([]string, 0, len(pods.Items))
+	ips := make([]string, 0, len(pods.Items)*2) //nolint
 	for index := range pods.Items {
-		ips = append(ips, pods.Items[index].Status.PodIP)
+		for _, podIP := range pods.Items[index].Status.PodIPs {
+			ips = append(ips, podIP.IP)
```

Comment:
Why do you want N*2 space allocated?

Reply:
Hi John, yes because here we might have 2 * times IPs in dualstack overlay scenario (each pod has two IPs, one is ipv4 and another is ipv6)